packages/turf-line-chunk/index.js

```diff
@@ -32,30 +31,21 @@ module.exports = function (featureIn, segmentLength, units, reverse) {
     var outFeatures = [];
     var debug = arguments['4']; // Hidden @param {boolean} Enable debug mode
 
-    // Handles FeatureCollection
-    featureEach(featureIn, function (multiFeature) {
-
-        // Handles MultiLineString
-        if (multiFeature.geometry.type === 'MultiLineString') {
-            multiFeature = flatten(multiFeature);
+    // Flatten each feature to simple LineString
+    flattenEach(featureIn, function (feature) {
+        if (reverse) {
+            feature.geometry.coordinates = feature.geometry.coordinates.reverse();
```

Comment:
I wonder if this would cause the input geojson to be mutated, I'll add a test case for that and see if this method causes input mutation.